build.sbt

```diff
@@ -102,7 +102,7 @@ ThisBuild / publishTo := {
 }
 
 // Sonatype settings
-ThisBuild / publishMavenStyle := true
+ThisBuild / publishMavenStyle.withRank(KeyRanks.Invisible) := true
```

Comment:
what is this for? not familiar...

Reply:
Remove the annoying warning that we got for years (This is the fix suggested by the warning).